buildconfig/config_msys2.py

```diff
@@ -356,7 +356,7 @@ def configure(self):
                     nonext_name = splitext(d.lib_dir)[0]
                     def_file = '%s.def' % nonext_name
                     basename = os.path.basename(nonext_name)
-                    print('Building lib from %s: %s.lib...' % (
+                    print('Building lib from {}: {}.lib...'.format(
```

Comment:
```suggestion
                    print('Building lib from {os.path.basename(d.lib_dir)}: {basename}.lib...')
```

Reply:
uh, not sure about the line break in my suggestion.

Reply:
ok, just read your comment above about pyupgrade to try to not harm readability. Not sure here.